dll/win32/shell32/wine/classes.c

```diff
@@ -46,6 +46,33 @@ WINE_DEFAULT_DEBUG_CHANNEL(shell);
 
 #define MAX_EXTENSION_LENGTH 20
 
+HRESULT HCR_GetProgIdKeyOfExtension(LPCWSTR szExtension, HKEY *phKey, BOOL AllowFallback)
```

Comment:
```suggestion
HRESULT HCR_GetProgIdKeyOfExtension(PCWSTR szExtension, PHKEY phKey, BOOL AllowFallback)
```
The 'L' in these "LP(C)WSTR" are not necessary at all.